qemu/tests/win_virtio_driver_update_by_installer.py

```diff
@@ -108,6 +137,12 @@ def change_virtio_media(cdrom_virtio):
                                                    expected_gagent_version)
     win_driver_installer_test.driver_check(session, test, params)
 
+    if reg_and_start_virtio_fs_svc:
+        win_driver_installer_test.viofs_svc_run(test, params, session)
+
+    if basic_io_test:
+        io_test(test, params, session)
+
     error_context.context("Run driver function test after update",
```

Comment:
Hope to add virtiofs function here,and combine with         test_func = "win_driver_installer_test.%s_test" % test_name .